spring-aot-test/src/main/java/org/springframework/aot/test/context/bootstrap/generator/TestContextAotProcessor.java

```diff
@@ -126,6 +132,34 @@ protected ClassName generateTestContext(BootstrapWriterContext writerContext, Su
 				.addModifiers(Modifier.PUBLIC, Modifier.STATIC).addCode(code.build());
 	}
 
+	private MethodSpec.Builder getContextInitializersBuilder(Map<ClassName, TestContextConfigurationDescriptor> entries) {
+		// We're generating a method that looks like the following.
```

Comment:
Please remove the comments. We now have a method that starts with an example while the existing one does not.

Reply:
I disagree with the rationale.

When JavaPoet is used to generate code, it is not readily clear what is being generated. Thus, the inclusion of the comment is beneficial to anyone reading the code and especially to anyone who has to modify or maintain the code.

Instead of removing this comment, I suggest we add a similar comment to `contextLoadersMappingMethod()` for consistency.

Reply:
Thanks but I still don't think this is a good idea, certainly for the reason you've shared. 

The tests are meant to show what is generated rather than a comment that is going to be outdated. If you need a visual representation of what the method will look like concretely, write a test for it that asserts the content. That's what do everywhere else.

Reply:
OK. I'll remove the comment.

I already have a test for it that asserts the content.